src/components/Question/Question.tsx

```diff
@@ -13,43 +13,59 @@ interface Props {
 export const Question: React.FC<Props> = ({ updateAllAnswers }) => {
 
   const questionContext = useContext(QuestionContext);
-  const questionKeys = Object.keys(questionContext )
+  const questionKeys = Object.keys(questionContext)
   const [answerInput, updateAnswer] = useState<any>({});
   const [index, setIndex] = useState<number>(0);
+  const[isSelected, setIsSelected] = useState<boolean>(false)
   const [errorMessage, setErrorMessage] = useState<string>("");
+  const[rentSelected, setRentSelected] = useState<boolean>(false);
+  const[principleSelected, setPrincipleSelected] = useState<boolean>(false)
   const inputRef = useRef<HTMLInputElement>(null);
```

Comment:
I liked the use of the state to know when the user has selected one of the other.